src/collective/ifttt/actions/ifttt.py

```diff
@@ -47,6 +47,8 @@ class IIftttTriggerAction(Interface):
             u'Give the name of IFTTT event which you want to trigger'
         ),
         required=True,
+        # BUG - add one word contraint to this and to all ifttt_triggers
+        # constraints=validate_ifttt_event,
```

Comment:
@Shriyanshagro Folloe https://docs.plone.org/develop/addons/schema-driven-forms/customising-form-behaviour/validation.html#constraints to add "constaint" to schema

constraint is a function that receives value to checked as its only argument (it knows nothing about the context of the value)

if values is ok it returs nothing

if value is not ok, it raises exception (see the exception class from the linked doc)